lib/main.js

```diff
@@ -69,11 +69,11 @@ Object.assign(exports, {
 
     this.reconnectTimeoutID = setTimeout(function() {
       metadataStore.asyncConnect().then(() => {connectRetried = 0;})
-      .catch(error => {
-        // increment the connect counter to avoid the endless retry
-        connectRetried++;
-        this.reconnectMetadataStore();
-      });
+        .catch(error => {
+          // increment the connect counter to avoid the endless retry
+          connectRetried++;
+          this.reconnectMetadataStore();
+        });
     }.bind(this), 500);
```

Comment:
Nope. Probably should do this as part of turning on http://eslint.org/docs/rules/prefer-arrow-callback than individually.